src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/MSBuildProjectManager.cs

```diff
@@ -109,11 +111,11 @@ public void RazorDocumentChanged(RazorFileChangeEventArgs args)
                 // When documents get added or removed we need to refresh project state to properly reflect the host documents in the project.
 
                 var evaluatedProjectInstance = _projectInstanceEvaluator.Evaluate(args.UnevaluatedProjectInstance);
-                Task.Factory.StartNew(
+                _ = Task.Factory.StartNew(
```

Comment:
Do we do anything smart to handle things if one of these bad boys throws an exception? I know that's not the point of this PR but I'm ready to be educated.

Reply:
It's a good question, we should probably be handling diagnostics; however, as it currently stands if something goes wrong we'll catastrophically explode resulting in the most severe of bugs that shouldn't go unnoticed 😄